src/testing/code-debugging.md

```diff
@@ -969,8 +1008,11 @@ setting the `showPerformanceOverlay` property to `true` on the
 [`MaterialApp`][], [`CupertinoApp`][], or [`WidgetsApp`][]
 constructor:
 
+#### Example 3
+{:.no_toc}
+
 <?code-excerpt "lib/performance_overlay.dart (PerfOverlay)" replace="/showPerformanceOverlay: true,/[[highlight]]$&[[\/highlight]]/g"?>
-{% prettify dart %}
+```dart
```

Comment:
Code highlighting doesn't yet working without prettify (I'm working on this now), so you'll have to switch this one back.

<img width="636" alt="The rendered output having the highlight tag" src="https://github.com/flutter/website/assets/18372958/304263a1-57e4-426f-a100-2c482e957458">

Reply:
Removed.